examples/seismic/elastic/wavesolver.py

```diff
@@ -41,41 +37,36 @@ def op_fwd(self, save=None):
         return ForwardOperator(self.model, save=save, geometry=self.geometry,
                                space_order=self.space_order, **self._kwargs)
 
-    def forward(self, src=None, rec1=None, rec2=None, vp=None, vs=None, rho=None,
-                vx=None, vz=None, txx=None, tzz=None, txz=None, save=None, **kwargs):
+    def forward(self, src=None, rec1=None, rec2=None, lam=None, mu=None, irho=None,
+                v=None, tau=None, save=None, **kwargs):
         """
         Forward modelling function that creates the necessary
         data objects for running a forward modelling operator.
 
         Parameters
         ----------
-        geometry : AcquisitionGeometry
-            Geometry object that contains the source (src : SparseTimeFunction) and
-            receivers (rec1(txx) : SparseTimeFunction, rec2(tzz) : SparseTimeFunction)
-            and their position.
-        vx : TimeFunction, optional
-            The computed horizontal particle velocity.
-        vz : TimeFunction, optional
-            The computed vertical particle velocity.
-        txx : TimeFunction, optional
-            The computed horizontal stress.
-        tzz : TimeFunction, optional
-            The computed vertical stress.
-        txz : TimeFunction, optional
-            The computed diagonal stresss.
-        vp : Function, optional
-            The time-constant P-wave velocity (km/s).
-        vs : Function, optional
-            The time-constant S-wave velocity (km/s).
-        rho : Function, optional
-            The time-constant density (rho=1 for water).
+        src : SparseTimeFunction or array_like, optional
+            Time series data for the injected source term.
+        rec1 : SparseTimeFunction or array_like, optional
+            The interpolated receiver data of the pressure (tzz).
+        rec2 : SparseTimeFunction or array_like, optional
+            The interpolated receiver data of the particle velocities.
+        v : VectorTimeFunction, optional
+            The computed  particle velocity.
+        tau : TensorTimeFunction, optional
+            The computed symmetric stress tensor.
+        lam : Function, optional
+            The time-constant first Lame parameter lambda.
+        mu : Function, optional
+            The time-constant second Lame parameter mu.
+        irho : Function, optional
+            The time-constant inverse density (irho=1 for water).
         save : int or Buffer, optional
             Option to store the entire (unrolled) wavefield.
 
         Returns
         -------
-        Rec1 (txx), Rec2 (tzz), particle velocities vx and vz, stress txx,
-        tzz and txz and performance summary.
+        Rec1(tzz), Rec2(div(v)), particle velocities v, stress t and performance summary.
```

Comment:
stresses?

Reply:
t->tau?

Reply:
Not stresses, the stress is one tensor called the stress tensor